Qast/Managers/StreamMetadataAPIManager.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  StreamAPI.swift
+//  Qast
+//
+//  Created by Andrew O'Brien on 7/22/19.
+//  Copyright © 2019 Qast. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+
+class UMGStreamMetadataAPIManager: StreamMetadataAPI {
+    
+    func streamMetadata(getCoverArtUrlforIsrc isrc: String, _ completion: @escaping (Result<URL>) -> Void) {
+        let urlString = "https://hackathon.umusic.com/prod/v1/isrc/\(isrc)/cover"
+        guard let url = URL(string: urlString) else {
+            return completion(.error(NSError(domain: "Error", code: 303, userInfo: nil)))
+        }
+        let headers = ["x-api-key": "5dsb3jqxzX8D5dIlJzWoTaTM2TzcKufq1geS1SSb"]
+        
+        Alamofire.request(url, method: .get, parameters: nil, encoding: JSONEncoding.default, headers: headers).responseObject { (response: DataResponse<CoverArtResponse>) in
+            switch response.result {
```

Comment:
Not great here how we switch first on the DataResult<Value> and then nest a switch block on our own Result<Value> .

I want to make DataRequest use our own completion handler if possible in future commits.